addons/main/functions/fnc_addShareInformationHandler.sqf

```diff
@@ -0,0 +1,27 @@
+/*
+ * Author: mjc4wilton
+ * Adds a custom handler to be called whenever the AI attempt to share information.
+ * Code should return a BOOL where true allows information to be passed and false blocks it.
+ * A single false will cause information to not be shared.
+ *
+ * Arguments:
+ * 0: function <CODE>
+ *
+ * Arguments passed to function:
+ * 0: unit sharing information <OBJECT>
+ * 1: enemy target <OBJECT>
+ * 2: range to share information, default 350 <NUMBER>
+ * 3: override radio ranges, default false <BOOLEAN>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [{false}] call lambs_main_fnc_addShareInformationHandler;
+ *
+ * Public: No
+*/
+
+params [["_code", {true}, [{}]]];
+
+GVAR(shareHandlers) pushBack _code;
```

Comment:
Where is `GVAR(shareHandlers)` being initialized?

Reply:
Knew I was forgetting something. Just pushed up a define in preInit.